stl/inc/memory

```diff
@@ -1788,8 +1850,9 @@ protected:
 };
 
 // CLASS TEMPLATE _Ref_count_obj_alloc3
-template <class _Ty, class _Alloc>
-class _Ref_count_obj_alloc3 : public _Ebco_base<_Rebind_alloc_t<_Alloc, _Ty>>, public _Ref_count_base {
+template <class _Ty, class _Alloc, _Init_form _Init>
+class __declspec(empty_bases) _Ref_count_obj_alloc3 : public _Ebco_base<_Rebind_alloc_t<_Alloc, _Ty>>,

```

Comment:
We recently removed `__declspec(empty_bases)` from all control blocks, after noticing that it didn't actually have any effect (only `_Ebco_base` might be empty; `_Ref_count_base` never is), and it was causing compiler errors with CUDA 9.2. The 3 occurrences of `__declspec(empty_bases)` being added here should be removed again.